REQUIRE

```diff
@@ -2,7 +2,7 @@ julia 0.4
 Compat 0.10.0
 URIParser 0.0.3
 @unix HTTPClient 0.0.0
-LibExpat 0.0.3
+LibExpat 0.2.8
```

Comment:
Minimum version bumped from 0.0.3 to 0.2.8 since this is the earliest version that support precompilation